### DISCUSSION TITLE
I really regret trying SvelteKit!!

### DISCUSSION THREAD

**Discussion by rolanday** [sveltejs/kit#8644] [Show and tell]
I was curious, and now I'm hooked after only one day -- it's like crack cocaine. I'm hitting pause on my latest project (a few months into it on Next13) and I will see how far I get with SvelteKit, but it looks good so far and I'm willing to give it a shot -- but its going to set me back in the short-term (the regret part).

SvelteKit has some growing pains, as to be expected for a relatively new framework (mainly, lack of middleware, mindshare), and sorely needs better debugger support (or documentation), but the dx is fantastic and even SvelteKit falls flat on its face (we hope not!!), the paradigm/approach to writing web apps will surely be adopted -- it makes react look really long in the tooth. Very exciting!

**Comment by multiplehats** [comment]
Good to hear you're hooked (pun intended).

Middleware can be created in +hooks.server.ts, did you look into that at all?
